.pre-commit-config.yaml

```diff
@@ -42,8 +42,4 @@ repos:
     hooks:
       - id: mypy
         pass_filenames: false
```

Comment:
This is leading me back to thinking it is a recursion problem. If I remove the pass_filename then both calls return errors, but they are not the same errors. On top of that there is no overlap whatsoever between the errors. 
Funny enough all of the pre-commit errors that are shown are errors that should not have been returned, because they are all contained within a test folder/file or in the docs. So its like it ran the opposite of what the exclude statement in the toml dictated